components/backdrop/backdrop.js

```diff
@@ -209,12 +209,14 @@ function showAccessible(elems) {
 		} else {
 			elem.removeAttribute('role');
 		}
-		const ariaHidden = elem.getAttribute('d2l-backdrop-aria-hidden');
-		if (ariaHidden) {
-			elem.setAttribute('aria-hidden', ariaHidden);
-			elem.removeAttribute('d2l-backdrop-aria-hidden');
-		} else {
-			elem.removeAttribute('aria-hidden');
+		if (elem.nodeName === 'FORM' || elem.nodeName === 'A') {
```

Comment:
Yep I'm curious why we only manipulate aria-hidden on these two types of nodes to begin with - but if that remains unchanged, then this fix makes sense.

Reply:
Yeah, I'd like to at least get this fixed, and then we can look at making a bigger change to this with a bit more investigation. Don't want to risk introducing a separate problem.

Reply:
Yeah I'd be tempted to apply this logic to all elements, not just forms and anchors. Our old Bitbucket server is down so I have a feeling we may never find the original pull request when the change was introduced.

Reply:
Cool, that's fair.